assets/data-port/stepper/index.jsx

```diff
@@ -0,0 +1,102 @@
+import classnames from 'classnames';
+
+/**
+ * Helper method to update the steps when moving to the next one.
+ *
+ * @param  {Array} steps  The current steps.
+ * @return {Array}        The steps after moving.
+ */
+const moveToNext = ( steps ) => {
+	const newSteps = [ ...steps ];
+
+	for ( let i = 0; i < newSteps.length; i++ ) {
+		if ( newSteps[ i ].isActive ) {
+			newSteps[ i ].isComplete = true;
+
+			if ( i + 1 < newSteps.length ) {
+				newSteps[ i + 1 ].isActive = true;
+				newSteps[ i ].isActive = false;
+			}
+
+			return newSteps;
+		}
+	}
+
+	throw new Error( 'No active step.' );
+};
+
+/**
+ * Helper method to update the steps when the active step is completed.
+ *
+ * @param  {Array} steps  The current steps.
+ * @return {Array}        The steps after completing the current one.
+ */
+const completeCurrentStep = ( steps ) => {
+	const newSteps = steps.map( ( step ) => {
+		if ( step.isActive ) {
+			step.isComplete = true;
+		}
+
+		return step;
+	} );
+
+	return newSteps;
+};
+
+/**
+ * Get the key of the current active step.
+ *
+ * @param  {Array} steps  The current steps.
+ * @return {string}       The key of the active step.
+ */
+const getCurrentStep = ( steps ) => {
+	for ( const step of steps ) {
```

Comment:
This selector is a good strategy! =)
But to be simpler, we could use the `steps.find( ( step ) => step.isActive )`.

Reply:
6c714d6f3509b1b7f923e371f52c7ef01c4d7119